### ISSUE TITLE
Organization Pages: Style

### ISSUE THREAD

**Issue by imohkay** [klpdotorg/dubdubdub#405]
NOTE: Always merge donation_frontend branch before starting
Login:
/organisation/1
/organisation/1/edit

**Comment by imohkay** [comment]
@batpad when I try to submit a new volunteer activity here (http://dev.klp.org.in/organisation/1/volunteer_activity), nothing happens (i.e. no validation or UI indicator). I presume this is not ready? The reason I ask is to know if all the fields are required? (for styling reasons).

**Comment by imohkay** [comment]
@batpad closing this. we can open a new issue for donations later, if required, since the donate tab link returns a 404.
